test/models/transaction_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class TransactionTest < ActiveSupport::TestCase
+   test "is valid" do
+     trans = Transaction.new(invoice_id: 1, credit_card_number: 1234123412341234, result: "success")
```

Comment:
Line is too long. [100/80]<br>Separate every 3 digits in the integer portion of a number with underscores(_).